.github/workflows/start_full_benchmarks.yml

```diff
@@ -25,8 +25,8 @@ jobs:
     strategy:
       matrix:
         command: [ boolean_bench, shortint_full_bench,
-                   integer_full_bench, signed_integer_full_bench, integer_gpu_full_bench,
-                   core_crypto_bench, core_crypto_gpu_bench, wasm_client_bench ]
+                   integer_full_bench, signed_integer_full_bench,
+                   core_crypto_bench, wasm_client_bench ]
```

Comment:
Are the GPU benchmarks triggered in integer_full_bench? I don't see the GPU benchmarks executed in `start_benchmarks` so if they're not here either we wouldn't be triggering them ever? :thinking:

Reply:
No, now they are standalone, their scheduled on their own. So there is no need to go through these starter workflows anymore.